src/applications/disability-benefits/all-claims/config/4192/index.js

```diff
@@ -2,16 +2,23 @@ import _ from 'lodash';
 import environment from '../../../../../platform/utilities/environment';
 
 import {
-  pastEmploymentFormIntro,
   pastEmploymentFormDownload,
+  pastEmploymentFormIntro,
+  pastEmploymentFormUpload,
 } from '../../pages';
 
 import { needsToEnterUnemployability } from '../../utils';
 
 // const showFormTutorial = formData => _.get(formData, 'view:upload4192Choice.view:4192Info', false);
+
 const isDownloading = formData =>
-  _.get(formData, 'view:upload4192Choice.view:download4192', false);
-// const isUploading = formData => _.get(formData, 'view:upload4192Choice.view:upload4192', false);
+  _.get(formData, 'view:upload4192Choice.view:download4192', false) &&
+  needsToEnterUnemployability(formData);
+
+const isUploading = formData =>
+  _.get(formData, 'view:upload4192Choice.view:upload4192', false) &&
+  needsToEnterUnemployability(formData);
+
```

Comment:
Should these be in utils?

Reply:
figured with the eventual logic that will have to go into the index made more sense to put them here, also they are only going to be used here